ide/extbrowser/nbproject/project.properties

```diff
@@ -1,26 +1,26 @@
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-
-javadoc.arch=${basedir}/arch.xml
-# test.unit.cp.extra and/or test.unit.run.cp.extra
-javac.source=1.8
-release.external/extbrowser-dlls-18.03.15.zip!/extbrowser.dll=modules/lib/extbrowser.dll
-release.external/extbrowser-dlls-18.03.15.zip!/extbrowser64.dll=modules/lib/extbrowser64.dll
-test.config.stableBTD.includes=**/*Test.class
-test.config.stableBTD.excludes=\
-    **/ExtWebBrowserTest.class,\
-    **/NbDdeBrowserImplTest.class
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+javadoc.arch=${basedir}/arch.xml
+# test.unit.cp.extra and/or test.unit.run.cp.extra
+javac.source=11
+release.external/extbrowser-dlls-18.03.15.zip!/extbrowser.dll=modules/lib/extbrowser.dll
+release.external/extbrowser-dlls-18.03.15.zip!/extbrowser64.dll=modules/lib/extbrowser64.dll
+test.config.stableBTD.includes=**/*Test.class
+test.config.stableBTD.excludes=\
+    **/ExtWebBrowserTest.class,\
+    **/NbDdeBrowserImplTest.class

```

Comment:
this diff indicates a line ending issue. Did you edit this outside of NetBeans?

there is also a git setting for that which might be the cause too:
https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings

Reply:
that's strange, I'm sure I changed it from within netbeans